### ISSUE TITLE
Automate last updated field

### ISSUE THREAD

**Issue by dougmet** [datasciencecampus/sdg-indicators#2373]
Previously "release date".

Can we use the file time stamp, or git, to decide when something was last updated?

@AnnCorp @MaxEngledewAdmin should this only refer to the data or also the metadata?

**Comment by MaxEngledewAdmin** [comment]
@dougmet after discussion with the team- it should refer to any update to that webpage at all, metadata or data, with the ambition that is should link the the relevant closed pull request for complete transparency over what has changed

**Comment by dougmet** [comment]
oh that would be a nice touch. Should be doable. I'll think about that.

**Comment by dougmet** [comment]
Note to self: Probably we'd remove last updated from the metadata and it would be added in at build time.  You can get individual file history at:

```
git log -n 1 -- _indicators/4-2-1.md
```

and 

```
git log -n 1 -- data/indicator_4-2-1.csv
```

and possibly some way that's easier to use in a variable. At the very least we can form a URL to the commit on GitHub with a combination of the `site` variables and commit ID.

**Comment by dougmet** [comment]
So link for the metadata would be `https://github.com/datasciencecampus/sdg-indicators/commit/810af049ea3add9eadfe6456ca42d0f2b146c05e` all of which is accessible.

**Comment by dougmet** [comment]
I think this is doable, but if time runs short we can always link to the file history on GitHub:

e.g. https://github.com/datasciencecampus/sdg-indicators/commits/develop/_indicators/4-2-1.md

which we can easily do from Jekyll. The last updated and commit ID will need a pre-processor. This is all possible but it will need to update the metadata file as part of the build, which is new.

**Comment by dougmet** [comment]
Last commit:
```sh
cd _indicators
git log -n 1 --pretty=format:%H --  1-1-1.md
> d8d4ede41e81bcfe52aadc245db8cad2989f357f
```

Time stamp:
```
git log -n 1 --pretty=format:%ai --  1-1-1.md
> 2018-03-23 15:03:20 +0000
```

**Comment by dougmet** [comment]
bash route

```sh
cat 1-1-1.md | sed "s|target_id:.*$|target_id: $PWD|g"
```

**Comment by dougmet** [comment]
Working up a bash script to do this.Wondering if we might need a few metadata fields to support shas and dates.

**Comment by dougmet** [comment]
I have a working script for this with local build (see below)
![image](https://user-images.githubusercontent.com/5878305/37963399-098a395c-31b6-11e8-91dc-8e6131100832.png)

The URLs point to the relevant commit on GitHub. Some major things to note:

1. The script has to edit *in-place* the indicator .md file. I'd prefer not to do this in-place. Rather I'd like to build from one directory and write out to another.
2. The script takes 2 minutes to run. Not a deal breaker but it further slows down the build.
3. I've started refactoring the code but I think this should be done all at once #2594 

All in all I've proved it's possible but we need this to fit in with the system in a less destructive way. If we go ahead with splitting the repos then this offers a nice opportunity to pull the data from an external repo and build it here.
